src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -333,20 +325,20 @@ default String checkAlertEquals(String expectedAlertText, double waitFor, double
      * @param timeTook             - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: if the alert is present, the text of the alert
      */
-    default String checkAlertMatches(String expectedAlertPattern, double waitFor, double timeTook) {
+    String checkAlertMatches(String expectedAlertPattern, double waitFor, double timeTook) {
         // check for the object to be present
         String alert;
-        boolean isAlertPresent = getApp().is().alertPresent();
+        boolean isAlertPresent = this.app.is().alertPresent();
         if (isAlertPresent) {
-            alert = getApp().get().alert();
+            alert = this.app.get().alert();
         } else {
-            getReporter().fail("to find alert with the text matching pattern <b>" + expectedAlertPattern + ON_PAGE, waitFor, NO_ALERT, timeTook);
+            this.reporter.fail("to find alert with the text matching pattern <b>" + expectedAlertPattern + ON_PAGE, waitFor, NO_ALERT, timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "to find alert with the text matching pattern <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)